locations/spiders/tesla.py

```diff
@@ -4,104 +4,60 @@
 import scrapy
 
 from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
 from locations.items import Feature
 
 
 class TeslaSpider(scrapy.Spider):
     name = "tesla"
     item_attributes = {"brand": "Tesla", "brand_wikidata": "Q478214"}
-    allowed_domains = ["www.tesla.com"]
-    start_urls = [
-        "https://www.tesla.com/findus/list",
-    ]
+
     download_delay = 0.5
     custom_settings = {
         "USER_AGENT": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36",
     }
```

Comment:
Is the custom useragent is still needed, can `BROWSER_DEFAULT` be used? If it's not needed, let's remove it.

Reply:
Seems to be needed